ui/dashboards/src/views/ViewDashboard/ViewDashboard.stories.tsx

```diff
@@ -0,0 +1,117 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import type { Meta, StoryObj } from '@storybook/react';
+import { ViewDashboard, DashboardProvider, EmptyDashboard } from '@perses-dev/dashboards';
+import { WithPluginRegistry, WithQueryParams, WithQueryClient } from '../../stories/decorators';
+
+const meta: Meta<typeof ViewDashboard> = {
+  component: ViewDashboard,
+  decorators: [WithQueryParams, WithPluginRegistry, WithQueryClient],
+  parameters: {
+    // Overriding the default on* regex for actions becaues we expose a LOT
+    // of these by exposing the MUI BoxProps, and it was making the storybook
+    // and browser hang from the numerous actions happening when you interacted
+    // with the page.
```

Comment:
We should probably take a pass at cleaning some of these MUI props, but that's technically a breaking change, so I don't want to do it in this PR.